ui/app/src/Utils/Views.elm

```diff
@@ -61,8 +65,8 @@ textField labelText content msg =
 
 buttonLink : String -> String -> String -> msg -> Html msg
 buttonLink icon link color msg =
-    a [ class <| "f6 link br1 ba mr1 ph3 pv2 mb2 dib " ++ color, href link, onClick msg ]
-        [ i [ class <| "fa fa-3 " ++ icon ] []
+    a [ class <| "" ++ color, href link, onClick msg ]
```

Comment:
Thanks for getting rid of the tachyons css classes, we should eventually rule it out